R/common.R

```diff
@@ -536,10 +536,18 @@ jaspResultsStrings <- function() {
 
 #' @export
 .extractErrorMessage <- function(error) {
+  stopifnot(length(error) == 1)
 
-  split <- base::strsplit(as.character(error), ":")[[1]]
-  last <- split[[length(split)]]
-  trimws(last)
+  if (isTryError(error)) {
+    msg <- attr(error, "condition")$message
+    return(trimws(msg))
+  } else if (is.character(error)){
+    split <- base::strsplit(error, ":")[[1]]
+    last <- split[[length(split)]]
+    return(trimws(last))
+  } else {
+    stop("Do not know what to do with an object of class `", class(error)[1], "`; The class of the `error` object should be `try-error` or `character`!" )
```

Comment:
```suggestion
    stop("Do not know what to do with an object of class `", class(error)[1], "`; The class of the `error` object should be `try-error` or `character`!", domain = NA)
```
otherwise translators will be asked to translate this message :)

Reply:
good to know!